app/templates/admin/messages.hbs

```diff
@@ -1 +1,58 @@
-{{outlet}}
+<div class="ui grid stackable">
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui stackable very basic table compact">
+        <thead>
+          <tr>
+            <th>{{t 'Recipient'}}</th>
+            <th>{{t 'Trigger'}}</th>
+            <th>{{t 'Email Message'}}</th>
+            <th>{{t 'Notification Message'}}</th>
+            <th>{{t 'Options'}}</th>
+            <th>{{t 'Sent At'}}</th>
```

Comment:
Please check the screen shot attached. The name of this column `Time/Date of Sent out`.